### ISSUE TITLE
Use yt to toggle between the 3D potential map and the CMB sphere

### ISSUE THREAD

**Issue by LaurencePeanuts** [rogerblandford/Music#26]
Add a method to the Universe class that makes a movie in yt where the 3D potential map fades away and the CMB sphere brightens with the potential fluctuations projected on it, with the same color map as the 3D potential.

**Comment by LaurencePeanuts** [comment]
here is a first attempt at this!
![gifortalk2](https://cloud.githubusercontent.com/assets/13594101/13531965/fadef976-e1dd-11e5-93f8-0cabe7efa17b.gif)

**Comment by rogerblandford** [comment]
Hi
That’s a nice job.
Are you talking about this today, Laurence?
I managed last night to combine the SW and ISW into a single expression - its actually easier this way.
I have not added in the velocity and density terms yet which kick in at l~ 30 I believe.
Roger
On Mar 4, 2016, at 7:53 AM, Laurence Perreault Levasseur <notifications@github.com<mailto:notifications@github.com>> wrote:

here is a first attempt at this!
[gifortalk2]https://cloud.githubusercontent.com/assets/13594101/13531965/fadef976-e1dd-11e5-93f8-0cabe7efa17b.gif

**Comment by LaurencePeanuts** [comment]
Hi Roger,

Yes I'm planning on briefly talk about this, but I'll also talk about
different things I've done in the past, like stochastic inflation.

That's great news for the ISW!

Best,
Laurence

2016-03-04 8:02 GMT-08:00 Roger Blandford notifications@github.com:

> Hi
> That’s a nice job.
> Are you talking about this today, Laurence?
> I managed last night to combine the SW and ISW into a single expression -
> its actually easier this way.
> I have not added in the velocity and density terms yet which kick in at l~
> 30 I believe.
> Roger
> On Mar 4, 2016, at 7:53 AM, Laurence Perreault Levasseur <

**Comment by drphilmarshall** [comment]
Hey Laurence, I thought your tea talk was excellent - and that gif is
great! My only suggestions are that you:

1) make it loop back and forth between showing the sphere and 3D potential
views

2) experiment with having a 3D potential only view, without the sphere at
all (this is less important than the looping though.

Great stuff!

**Comment by LaurencePeanuts** [comment]
Thanks Phil!

Here is 1) for the gif I showed above. I'm still not completely happy with it because it's still a bit too jumpy, but it gives an idea
![loop](https://cloud.githubusercontent.com/assets/13594101/13615013/2709a652-e528-11e5-8ef7-2c7d693745fb.gif)

For 2) I'm not sure what you mean...

**Comment by rogerblandford** [comment]
Looks good

Sent from my iPhone

On Mar 8, 2016, at 12:22, Laurence Perreault Levasseur <notifications@github.com<mailto:notifications@github.com>> wrote:

Thanks Phil!

Here is 1) for the gif I showed above. I'm still not completely happy with it because it's still a bit too jumpy, but it gives an idea
[loop]https://cloud.githubusercontent.com/assets/13594101/13615013/2709a652-e528-11e5-8ef7-2c7d693745fb.gif

For 2) I'm not sure what you mean...

##

**Comment by drphilmarshall** [comment]
Agreed - you did the "3D - only view" that  I was asking for in 2) as well!
Probably it was already there and I missed it :-)
